src/Avalonia.Controls/NativeMenu.cs

```diff
@@ -0,0 +1,76 @@
+﻿using System;
+using System.Collections;
+using System.Collections.Generic;
+using System.Reactive.Linq;
+using Avalonia.Collections;
+using Avalonia.Controls.Platform;
+using Avalonia.LogicalTree;
+using Avalonia.Metadata;
+
+namespace Avalonia.Controls
+{
+    public class NativeMenu : Control
```

Comment:
@danwalmsley we talked about this in skype, but the question was: does it make sense for `NativeMenu` to be a control? It's never actually going to be displayed as a control in a window. I think one of the options we discussed was to have it as a non-control exposed via e.g. `Window.NativeMenu`.

Did you decide to not go with that approach? I'd be interested to hear why for future reference.

Reply:
The current plan is to move native menu items to `NativeMenuBar.Items` attached property on the `Window` instead and then have `ManagedMenuBarPresenter` that would render our managed menu if native support is missing, so the usage would be something like:

```xml
<Window NativeMenuBar.Items="{Binding MenuItems}">
   <DockPanel>
       <ManagedMenuBarPresenter DockPanel.Dock="Top"/>
       <!-- the rest of the window -->
   </DockPanel>
</Window>
```